### PR TITLE
Updated DATAPROFILER_SEED setting in utils.py

### PR DESCRIPTION
**Pull request by jacob-buehler**
this should fix issue #916 https://github.com/capitalone/DataProfiler/issues/916

### CODE REVIEW

**Review by taylorfturner — CHANGES_REQUESTED** [review]
we'll want some validation / testing of this in `dataprofiler/tests/test_utils.py`

**Comment by jacob-buehler** [comment]
> we'll want some validation / testing of this in `dataprofiler/tests/test_utils.py`

OK I can do this

File: dataprofiler/profilers/utils.py [hunk 1]
```diff
@@ -124,7 +122,9 @@ def shuffle_in_chunks(
 
         # Generate random list of indexes
         lower_bound_list = np.array(range(j, j + true_chunk_size))
-        random_list = rng.integers(lower_bound_list, data_length)
+        random_list = np.empty(len(lower_bound_list))
+        for i in range(len(lower_bound_list)):
+            random_list[i] = rng.randint(lower_bound_list[i], data_length)
```
Comment (JGSweets) [inline dataprofiler/profilers/utils.py:127]: Is this change needed? Or why was it needed? I'm concerned that looping as well is going to be expensive.

**Comment by JGSweets** [comment]
@taylorfturner also mentioned also abstracting usage for where this is used in the library.

**Comment by jacob-buehler** [comment]
I am closing this PR, and am moving all work to rngimpo branch on my personal fork

**Pull request closed without merging**